File: MPI/mpicheck/tests/functions/2.2/MPI_Dist_graph_create.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2(10)
       INTEGER var_3(10)
       INTEGER var_4(10)
       INTEGER var_5(10)
       INTEGER var_6
       LOGICAL var_7
       INTEGER var_8
       INTEGER var_9
        call mpi_dist_graph_create(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, &
       var_9)
       call pmpi_dist_graph_create(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, &
       var_9)
        end program main
    

File: MPI/mpicheck/tests/functions/2.2/MPI_Dist_graph_create.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    int var_1;
    const int var_2[2];
    const int var_3[2];
    const int var_4[2];
    const int var_5[2];
    MPI_Info var_6;
    int var_7;
    MPI_Comm *var_8;
    int ret;
    /* calls */
    ret = MPI_Dist_graph_create(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8);
    ret = PMPI_Dist_graph_create(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8);
    return 0;
}
